zetaclient/zetabridge/tx.go

```diff
@@ -151,19 +151,19 @@ func (b *ZetaCoreBridge) SetTSS(tssPubkey string, keyGenZetaHeight int64, status
 	return "", fmt.Errorf("set tss failed | err %s", err.Error())
 }
 
-func (b *ZetaCoreBridge) ConfigUpdater(cfg *config.Config) {
-	b.logger.Info().Msg("ConfigUpdater started")
-	ticker := time.NewTicker(time.Duration(cfg.ConfigUpdateTicker) * time.Second)
+func (b *ZetaCoreBridge) CoreContextUpdater(appContext *appcontext.AppContext) {
```

Comment:
```suggestion
func (b *ZetaCoreBridge) UpdateCoreContext(appContext *appcontext.AppContext) {
```
The original nameing is wrong this describe action. `Updater` would be used for the type.

Also shouldn't be `UpdateAppContext` instead?